daemon/api_test.go

```diff
@@ -202,6 +202,7 @@ func (s *apiSuite) TestSnapInfoOneIntegration(c *check.C) {
 			"update_available":   "v2",
 			"rollback_available": "v0",
 			"channel":            "stable",
+			"price":              float64(0),
```

Comment:
Isn't there a decimal type that could be used instead. Using floats for currency is a great way to lose (or make) money.